app/controllers/users/details.js

```diff
@@ -47,7 +53,42 @@ export default Ember.Controller.extend(OrganisationMixin, {
     }
   ),
 
+  disableUserAccount(disabled) {
+    if (this.get("canDisableUsers")) {
```

Comment:
would it be better to have any error message displayed, if user dont have this permission ?

Reply:
The link to disable/enable account are only accessible to users who have permission to do it, otherwise, it will be always a hidden link.